tests/unit/backend/utils.py

```diff
@@ -60,10 +60,20 @@ def get(cls, **kwargs):
 
 class BogusGenesetParams:
     @classmethod
-    def get(cls, **kwargs):
+    def get(cls, gene_count=6, **kwargs):
         genes = []
-        for i in range(6):
-            genes.append(cls.generate_random_string(4))
+        for i in range(gene_count):
+            gene = {
+                "gene_symbol": cls.generate_random_string(4),
```

Comment:
Mitigate the chance of having a flakey test by making this more predictablel. Maybe just an incrementer.